### PR TITLE
[DOC-8083] Updates to Alerts page for Azure

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-8083] Updates to Alerts page for Azure

- Small general improvements
- Temporarily remove CMEK alert for encrypted backup until it works
- No Azure-specific changes

## Previews
[src/current/cockroachcloud/alerts-page.md](https://deploy-preview-17328--cockroachdb-docs.netlify.app/docs/cockroachcloud/alerts-page.html)

## Tests
- [x] Local build
- [x] Linkcheck

### CODE REVIEW

**Review by ianjevans — APPROVED** [review]

File: src/current/cockroachcloud/alerts-page.md [hunk 1]
```diff
@@ -3,15 +3,16 @@ title: Alerts Page
...
+The **Alerts** page allows you to enable email alerts, send test alerts, and view the email recipients and alert history for your {{ site.data.products.db }} organization. To view the Alerts page, [log in](https://cockroachlabs.cloud/) and click **Alerts**.
 
 {{site.data.alerts.callout_info}}
-The **Alerts** page is accessible on {{ site.data.products.dedicated }} clusters. For {{ site.data.products.serverless }} clusters, all [Org Administrators](authorization.html#org-administrator-legacy) will automatically receive email alerts when your cluster reaches 50%, 75%, and 100% of your [resource limits](../{{site.versions["stable"]}}/architecture/glossary.html#resource-limits).
+The **Alerts** page is accessible on {{ site.data.products.dedicated }} clusters. For {{ site.data.products.serverless }} clusters, all [Org Administrators](authorization.html#org-administrator-legacy) automatically receive email alerts when your cluster reaches 50%, 75%, and 100% of your [resource limits](../{{site.versions["stable"]}}/architecture/glossary.html#resource-limits).
```
Comment (abhinavg6) [inline src/current/cockroachcloud/alerts-page.md:12]:
```suggestion
The **Alerts** page is applicable for {{ site.data.products.dedicated }} clusters in your {{ site.data.products.db }} organization. For {{ site.data.products.serverless }} clusters in your organization, all [Org Administrators](authorization.html#org-administrator-legacy) automatically receive email alerts when your cluster reaches 50%, 75%, and 100% of your [resource limits](../{{site.versions["stable"]}}/architecture/glossary.html#resource-limits).
```

File: src/current/cockroachcloud/alerts-page.md [hunk 2]
```diff
@@ -31,24 +32,30 @@ If alerts are enabled, {{ site.data.products.db }} will send alerts to [specifie
 **CMEK:**
 
 - Cluster node unable to start due to CMEK key access failure.
-- Encrypted backup failed due to CMEK key access failure.
+{% comment %}- Encrypted backup failed due to CMEK key access failure.{% endcomment %}
 
 If you receive an alert repeatedly, you may need to [optimize your workload](../stable/make-queries-fast.html) or [scale your {{ site.data.products.db }} cluster](cluster-management.html?filters=dedicated#add-or-remove-nodes-from-a-cluster).
 
-[Org Administrators](authorization.html#org-administrator-legacy) will also receive email alerts when your cluster undergoes an automatic [patch version upgrade](upgrade-policy.html#patch-version-upgrades).
+[Org Administrators](authorization.html#org-administrator-legacy) also receive email alerts when your cluster undergoes an automatic [patch version upgrade](upgrade-policy.html#patch-version-upgrades).
```
Comment (abhinavg6) [inline src/current/cockroachcloud/alerts-page.md:39]: I'm checking with folks if this is true. News for me.

File: src/current/cockroachcloud/alerts-page.md [hunk 3]
```diff
@@ -31,24 +32,30 @@ If alerts are enabled, {{ site.data.products.db }} will send alerts to [specifie
...
 
 {{site.data.alerts.callout_success}}
-When scaling your cluster, we recommend first scaling VMs to include more than 2 vCPUs each. If this doesn't sufficiently improve performance, then add more nodes.
+When scaling your cluster, Cockroach Labs recommends that VMs for production clusters include a minimum of {% include {{ page.page_version }}/prod-deployment/provision-cpu.md threshold='minimum' %}. If increasing vCPU doesn't sufficiently improve performance, then add more nodes. Refer to [Recommended Production Settings](/docs/{{site.versions["stable"]}}/recommended-production-settings.html).
```
Comment (abhinavg6) [inline src/current/cockroachcloud/alerts-page.md:42]: I think we should keep this update out of this PR, and I would like it to be a separate one which can be reviewed by relevant people.

File: src/current/cockroachcloud/alerts-page.md [hunk 4]
```diff
@@ -31,24 +32,30 @@ If alerts are enabled, {{ site.data.products.db }} will send alerts to [specifie
...
 
 To enable alerts:
 
-- Under **Configure {{ site.data.products.db }} alerts**, toggle the **Alerts are on** switch.
-- Under **Add Email Recipients**, add at least one email address and click **Add**.
+1. Under **Configure {{ site.data.products.db }} alerts**, toggle the **Alerts are on** switch.
+1. Under **Add Email Recipients**, add at least one email address and click **Add**.
 
-Note that alerts are enabled for all clusters in your Organization. To filter alerts on specific clusters, you can use an email alias to send alerts to a monitoring tool such as [Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/) or <a href="https://www.pagerduty.com/" data-proofer-ignore>PagerDuty</a>.
+Alerts are enabled for all clusters in your Organization.
```
Comment (abhinavg6) [inline src/current/cockroachcloud/alerts-page.md:52]: Dedicated only?

**Review by abhinavg6 — CHANGES_REQUESTED** [review]
Done one pass. Requested a couple of changes, and verifying one thing with folks.

File: src/current/cockroachcloud/alerts-page.md (hunk 3 above)
Comment (mdlinville) [reply src/current/cockroachcloud/alerts-page.md:42]: Discussed offline, this info doesn't belong in this page anyway. I'll remove it.

File: src/current/cockroachcloud/alerts-page.md (hunk 4 above)
Comment (mdlinville) [inline src/current/cockroachcloud/alerts-page.md:52]:
```suggestion
Alerts are enabled for all {{ site.data.products.dedicated }} clusters in your Organization.
```

**Review by abhinavg6 — APPROVED** [review]
LGTM, thank you.

**Pull request merged**